DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs

```diff
@@ -152,8 +152,14 @@ internal static LogLevel ApplicationCommandsLogLevel
 		/// </summary>
 		internal static bool CheckAllGuilds { get; set; }
 
+		/// <summary>
+		/// Gets whether the registration check should be manually overriden.
+		/// </summary>
 		internal static bool ManOr { get; set; }
```

Comment:
I never noticed this name before. Should we uh... call it ManualOverride or something. This member name is not super helpful. Granted the summary does help.

Reply:
It is really just internal. I don't want to expose it that much.